E3SM-2-0/check/proof

```diff
@@ -0,0 +1,120 @@
+../historical_r5i1p1f1.json:  "experiment_id": "historical",
+../piClim-control_r1i1p1f1.json:  "experiment_id": "piClim-control",
+../piClim-histaer_r1i1p1f1.json:  "experiment_id": "piClim-histaer",
+../piClim-histaer_r2i1p1f1.json:  "experiment_id": "piClim-histaer",
+../piClim-histaer_r3i1p1f1.json:  "experiment_id": "piClim-histaer",
+../piClim-histall_r1i1p1f1.json:  "experiment_id": "piClim-histall",
+../piClim-histall_r2i1p1f1.json:  "experiment_id": "piClim-histall",
+../piClim-histall_r3i1p1f1.json:  "experiment_id": "piClim-histall",
+../piControl_r1i1p1f1.json:  "experiment_id": "piControl",
+ 
+../1pctCO2_r1i1p1f1.json:  "history": "v2.LR.historical_0101",
+../abrupt-4xCO2_r1i1p1f1.json:  "history": "v2.LR.abrupt-4xCO2_0101",
+../abrupt-4xCO2_r2i1p1f1.json:  "history": "v2.LR.abrupt-4xCO2_0301",
+../amip_r1i1p1f1.json:  "history": "v2.LR.amip_0101",
+../amip_r2i1p1f1.json:  "history": "v2.LR.amip_0201",
+../amip_r3i1p1f1.json:  "history": "v2.LR.amip_0301",
+../hist-aer_r1i1p1f1.json:  "history": "v2.LR.hist-aer_0101",
+../hist-aer_r2i1p1f1.json:  "history": "v2.LR.hist-aer_0151",
+../hist-aer_r3i1p1f1.json:  "history": "v2.LR.hist-aer_0201",
+../hist-aer_r4i1p1f1.json:  "history": "v2.LR.hist-aer_0251",
+../hist-aer_r5i1p1f1.json:  "history": "v2.LR.hist-aer_0301",
+../hist-GHG_r1i1p1f1.json:  "history": "v2.LR.hist-GHG_0101",
+../hist-GHG_r2i1p1f1.json:  "history": "v2.LR.hist-GHG_0151",
+../hist-GHG_r3i1p1f1.json:  "history": "v2.LR.hist-GHG_0201",
+../hist-GHG_r4i1p1f1.json:  "history": "v2.LR.hist-GHG_0251",
+../hist-GHG_r5i1p1f1.json:  "history": "v2.LR.hist-GHG_0301",
+../historical_r1i1p1f1.json:  "history": "v2.LR.historical_0101",
+../historical_r2i1p1f1.json:  "history": "v2.LR.historical_0151",
+../historical_r3i1p1f1.json:  "history": "v2.LR.historical_0201",
+../historical_r4i1p1f1.json:  "history": "v2.LR.historical_0251",
+../historical_r5i1p1f1.json:  "history": "v2.LR.historical_0301",
+../piClim-control_r1i1p1f1.json:  "history": "v2.LR.piClim-control",
+../piClim-histaer_r1i1p1f1.json:  "history": "v2.LR.piClim-histaer_0021",
+../piClim-histaer_r2i1p1f1.json:  "history": "v2.LR.piClim-histaer_0031",
+../piClim-histaer_r3i1p1f1.json:  "history": "v2.LR.piClim-histaer_0041",
+../piClim-histall_r1i1p1f1.json:  "history": "v2.LR.piClim-histall_0021",
+../piClim-histall_r2i1p1f1.json:  "history": "v2.LR.piClim-histall_0031",
+../piClim-histall_r3i1p1f1.json:  "history": "v2.LR.piClim-histall_0041",
+../piControl_r1i1p1f1.json:  "history": "v2.LR.piControl",
+ 
+../1pctCO2_r1i1p1f1.json:  "realization_index": "1",
+../abrupt-4xCO2_r1i1p1f1.json:  "realization_index": "1",
+../abrupt-4xCO2_r2i1p1f1.json:  "realization_index": "2",
+../amip_r1i1p1f1.json:  "realization_index": "1",
+../amip_r2i1p1f1.json:  "realization_index": "2",
+../amip_r3i1p1f1.json:  "realization_index": "3",
+../hist-aer_r1i1p1f1.json:  "realization_index": "1",
+../hist-aer_r2i1p1f1.json:  "realization_index": "1",
```

Comment:
should fix "realization_index"

Reply:
This does not make sense.  I just ran my "checker" on it, and I got (for realization index):

../1pctCO2_r1i1p1f1.json:  "realization_index": "1",
../abrupt-4xCO2_r1i1p1f1.json:  "realization_index": "1",
../abrupt-4xCO2_r2i1p1f1.json:  "realization_index": "2",
../amip_r1i1p1f1.json:  "realization_index": "1",
../amip_r2i1p1f1.json:  "realization_index": "2",
../amip_r3i1p1f1.json:  "realization_index": "3",
../hist-aer_r1i1p1f1.json:  "realization_index": "1",
../hist-aer_r2i1p1f1.json:  "realization_index": "2",
../hist-aer_r3i1p1f1.json:  "realization_index": "3",
../hist-aer_r4i1p1f1.json:  "realization_index": "4",
../hist-aer_r5i1p1f1.json:  "realization_index": "5",
../hist-GHG_r1i1p1f1.json:  "realization_index": "1",
../hist-GHG_r2i1p1f1.json:  "realization_index": "2",
../hist-GHG_r3i1p1f1.json:  "realization_index": "3",
../hist-GHG_r4i1p1f1.json:  "realization_index": "4",
../hist-GHG_r5i1p1f1.json:  "realization_index": "5",
../historical_r1i1p1f1.json:  "realization_index": "1",
../historical_r2i1p1f1.json:  "realization_index": "2",
../historical_r3i1p1f1.json:  "realization_index": "3",
../historical_r4i1p1f1.json:  "realization_index": "4",
../historical_r5i1p1f1.json:  "realization_index": "5",
../piClim-control_r1i1p1f1.json:  "realization_index": "1",
../piClim-histaer_r1i1p1f1.json:  "realization_index": "1",
../piClim-histaer_r2i1p1f1.json:  "realization_index": "2",
../piClim-histaer_r3i1p1f1.json:  "realization_index": "3",
../piClim-histall_r1i1p1f1.json:  "realization_index": "1",
../piClim-histall_r2i1p1f1.json:  "realization_index": "2",
../piClim-histall_r3i1p1f1.json:  "realization_index": "3",
../piControl_r1i1p1f1.json:  "realization_index": "1",

and my "git status" is up to date with the branch "add_v2_again".